src/extension.ts

```diff
@@ -95,9 +97,12 @@ export async function activate(context: vscode.ExtensionContext) {
             if (!arduinoPath || !validateArduinoPath(arduinoPath, useArduinoCli)) {
                 Logger.notifyUserError("InvalidArduinoPath", new Error(constants.messages.INVALID_ARDUINO_PATH));
                 vscode.commands.executeCommand("workbench.action.openGlobalSettings");
-            } else if (!commandPath || !util.fileExistsSync(commandPath)) {
+            } else if (commandPath && !util.fileExistsSync(commandPath)) {
                 Logger.notifyUserError("InvalidCommandPath", new Error(constants.messages.INVALID_COMMAND_PATH + commandPath));
             } else {
+                if(commandPath) {
+                    execFileSync(commandPath);
```

Comment:
Why are we executing `commandPath` here? `registerArduinoCommand()` is called in many places, some of which [don't directly interact with the Arduino IDE](https://github.com/microsoft/vscode-arduino/blob/1d4e62ea7ffe2a97b6cde75a2cfd3f22040518cf/src/extension.ts#L120).